control-plane/agents/src/bin/core/controller/reconciler/persistent_store.rs

```diff
@@ -33,4 +36,12 @@ impl TaskPoller for PersistentStoreReconciler {
 
         PollResult::Ok(PollerState::Idle)
     }
+    async fn poll_event(&mut self, context: &PollContext) -> bool {
+        match context.event() {
+            PollEvent::TimedRun => true,
+            PollEvent::Triggered(PollTriggerEvent::SimulStart) => true,
```

Comment:
Should we use this just on debug buillds?

Reply:
We could though this may be needed on release builds, ex a problem we only hit on release builds